controllers/user/update-email.js

```diff
@@ -80,25 +62,24 @@ router
                     await sendActivationEmail(req, updatedUser, true);
```

Comment:
This will conflict with my branch but you just need to remove `true` – we previously showed a different message on the confirmation screen for new versus existing users, but it's the same now.